front/src/modules/pathfinding/helpers/getStepLocation.ts

```diff
@@ -1,8 +1,7 @@
 import type { PathItemLocation } from 'common/api/osrdEditoastApi';
-import type { PathStep, StdcmPathStep } from 'reducers/osrdconf/types';
 import { mToMm } from 'utils/physics';
 
-const getStepLocation = (step: PathStep | StdcmPathStep): PathItemLocation => {
+const getStepLocation = <T extends PathItemLocation>(step: T): PathItemLocation => {
```

Comment:
I don't think we need a generic here. I think this works as well?

```ts
const getStepLocation = (step: PathItemLocation): PathItemLocation => {
```

Reply:
Yes it works :) I was anticipating, but you're right, I should keep it simple